### PR TITLE
SIP23 update to pending status

### PR DESCRIPTION
**Pull request by soronpo**
Literal types were implemented in both Typelevel Scala and Dotty.
New author volunteered to update this SIP, and bring it to an accepted status after committee review.
Require approval to change SIP status from 'dormant' to 'pending'.

### CODE REVIEW

**Comment by soronpo** [comment]
FYI @milessabin @paulp  @jvican

File: sips/pending/_posts/2014-06-27-42.type.md [hunk 1]
```diff
@@ -3,18 +3,30 @@ layout: sip
...
+| Jul 15th 2014  | Last update to SIP before declared dormant                         |
+| TBD            | Dormant because original authors are not available for its review  |
+| Feb 9th 2017   | New author volunteered to update the SIP for review                |
+
+### Current status
+Literal types were implemented in both Typelevel Scala and Dotty.
+
+New author volunteered to update this SIP, and bring it to an accepted status after committee review.
+
+Require approval to change SIP status from 'dormant' to 'pending'.
```
Comment (jvican) [inline sips/pending/_posts/2014-06-27-42.type.md]: Can we remove this sentence?

**Comment by jvican** [comment]
I think we have hit a Travis timeout @soronpo, can you push again?

**Comment by SethTisue** [comment]
I think you'll have to rebase to get the Travis timeout fix

**Comment by jvican** [comment]
Hey @soronpo, can you rebase this once https://github.com/scala/scala.github.com/pull/684 is merged? Hopefully we'll merge your proposal.

**Comment by soronpo** [comment]
Yes. I'll repush both SIPs once this issue is resolve.

**Comment by jvican** [comment]
Can you please rebase on top of https://github.com/scala/scala.github.com/pull/685?

**Pull request merged**